app/models/user.rb

```diff
@@ -73,6 +73,8 @@ class User < ApplicationRecord
     :authenticatable_salt,
     :invited_to_sign_up?, to: :email_authentication, allow_nil: true
 
+  after_destroy :destroy_email_authentications
```

Comment:
How about :phone_number_authentications? Should we figure out a deletion strategy for non-admin users?

Reply:
We could/should, but I've chosen to keep this fix tactical, addressing just the use-case we understand thoroughly today.

On the surface, I'm less concerned with app users since we have a robust "Allow/Deny Sync" mechanism for em.